crates/apub/src/objects/post.rs

```diff
@@ -210,14 +210,15 @@ impl ApubObject for ApubPost {
         updated: page.updated.map(|u| u.naive_local()),
         deleted: Some(false),
         nsfw: page.sensitive,
-        stickied: page.stickied,
         embed_title,
         embed_description,
         embed_video_url,
         thumbnail_url,
         ap_id: Some(page.id.clone().into()),
         local: Some(false),
         language_id,
+        featured_community: page.stickied,
+        featured_local: Some(false),
```

Comment:
Its possible that a local admin features a remote post. In case the post creator edits the post, this would overwrite the value for featured. You should just set None here so it keeps the existing value.

Reply:
Okay fixed